src/cli/doc/Cli.md

```diff
@@ -0,0 +1,40 @@
+# CLI
+
+## Special Files and Folders
+There are several manipulations on the files of the template when packaging it and running it. Most files are copied as-is and modified by the renaming phase, but that's not true for everything.
```

Comment:
Whenever you do markdown files, you should put each sentence on a new line to ease reviews and changes.

Reply:
Updated.

Reply:
![image](https://github.com/nventive/FlutterApplicationTemplate/assets/39710855/0043bb46-10d1-47da-969b-058431c975b1)